ckan/templates/package/snippets/resource_view.html

```diff
@@ -37,26 +37,26 @@
           <p id="data-view-error" class="collapse"></p>
           <p>
             <a href="{{ resource.url }}" class="btn btn-default btn-lg resource-url-analytics" target="_blank">
-		<i class="fa fa-lg fa-arrow-circle-o-down"></i>
-		{{ _('Download resource') }}
+		          <i class="fa fa-lg fa-arrow-circle-o-down"></i>
+		          {{ _('Download resource') }}
```

Comment:
mixing tabs and spaces?

Reply:
Nice catch. We are using `.editorconfig` file, that sets indentation to 2 spaces, so this file wasn't updated a long time if there still tabs here. There are few other places with mixed spaces and tabs - I won't change those files unless I need to edit them in the scope of current PR. Otherwise, diffs are unnecessary big, because GitHub cannot notice that the only thing that was changed is indentation:(